src/vmm/src/lib.rs

```diff
@@ -157,8 +165,12 @@ impl TryFrom<VMMConfig> for VMM {
             kernel_cfg: config.kernel_config.clone(),
         };
 
-        vmm.add_serial_console()?;
         vmm.create_vcpus(&config.vcpu_config)?;
+        vmm.add_serial_console()?;
+
+        if let Some(block_cfg) = config.block_config.as_ref() {
+            vmm.add_block_device(block_cfg)?;
```

Comment:
Can you also add a test for `TryFrom` where you also pass a block device configuration?

Reply:
You mean a new test, or is there some existing test somewhere?